FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -147,6 +160,34 @@ - (FIRTrace *)activeTrace {
   return self.backgroundSessionTrace;
 }
 
+- (void)startTrackingNetwork {
+  self.networkType = firebase_perf_v1_NetworkConnectionInfo_NetworkType_NONE;
+
+  if (@available(iOS 12, *)) {
+    dispatch_queue_attr_t attrs = dispatch_queue_attr_make_with_qos_class(
+        DISPATCH_QUEUE_SERIAL, QOS_CLASS_UTILITY, DISPATCH_QUEUE_PRIORITY_DEFAULT);
+    self.monitorQueue = dispatch_queue_create("com.example.network.monitor", attrs);
```

Comment:
More descriptive name for the queue?

Reply:
Good catch. Made the change, did not push it. Now, done.